airbyte_cdk/sources/file_based/file_types/excel_parser.py

```diff
@@ -181,14 +182,84 @@ def validate_format(excel_format: BaseModel, logger: logging.Logger) -> None:
             raise ConfigValidationError(FileBasedSourceError.CONFIG_VALIDATION_ERROR)
 
     @staticmethod
-    def open_and_parse_file(fp: Union[IOBase, str, Path]) -> pd.DataFrame:
+    def open_and_parse_file(
+        fp: Union[IOBase, str, Path],
+        logger: Optional[logging.Logger] = None,
+        file_info: Optional[Union[str, RemoteFile]] = None,
```

Comment:
Is it possible to have to `file_info` not a type of `RemoteFile`?
Also I think we can remove `Optional` from typing for `logger` and `file_info`, because in both times where we call it we've already added these arguments.